readme.md

```diff
@@ -57,6 +57,44 @@ If using TypeScript, don't forget to add 'babylonjs' to 'types' in tsconfig.json
 
 To add a module install the respected package. A list of extra packages and their installation instructions can be found on [babylonjs' user at npm](https://www.npmjs.com/~babylonjs).
 
+## Usage

```

Comment:
wondering if we should point people to this doc instead: http://doc.babylonjs.com/ (and perhaps just add an anchor to the get started portion of it)

Reply:
And perhaps mention how to import

Reply:
Sure, how about this? I still am showing code but I can remove that too if you think it adds too much clutter, but I think that for a lot of developers github is the first and often only place they may land and showing code strait away is a good way to get them hooked.

Reply:
Nope this is fine :) As long as we point to the doc as well like you did